guides/common/modules/proc_configuring-how-many-processes-can-be-launched-by-apache-httpd.adoc

```diff
@@ -0,0 +1,22 @@
+[id="configuring_how_many_processes_can_be_launched_by_apache_httpd_{context}"]
+= Configuring how many processes can be launched by Apache httpd
+
+.Configuring how many processes can be launched by Apache httpd
+
+By default, HTTPD uses event request handling mechanism.
+
+When the number of requests to the apache exceed the maximum number of child processes that can be launched to handle the incoming connections, an HTTP 503 Service Unavailable Error is raised by httpd.
+Amidst httpd running out of processes to handle, the incoming connections can also result in multiple component failure on your Foreman side due to the dependency of components like Pulp on the availability of httpd processes.
+
+You can adapt the configuration of HTTPD event to handle more concurrent requests based on your expected peak load.
+
+. You can modify these values in /etc/foreman-installer/custom-hiera.yaml by changing or adding these lines:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+apache::mod::event::serverlimit: 64
+apache::mod::event::maxrequestworkers: 1024
+apache::mod::event::maxrequestsperchild: 4000
+----
++
+These are the numbers installer will apply when using option --tuning=medium.
```

Comment:
I'm going to say that this needs a big disclaimer that it breaks the tuning functionality. The above values are what the installer sets with `--tuning medium` (https://github.com/theforeman/foreman-installer/blob/develop/config/foreman.hiera/tuning/sizes/medium.yaml) and above. If users place these values in `custom-hiera.yaml` we can never override them again.

Reply:
Hello @ekohl : I can understand your point and totally agree on this but we got a request from customers to update this section soon as mentioned in this bug - https://bugzilla.redhat.com/show_bug.cgi?id=2158458

As performance testing is still in progress (we cannot propose any number as of now), so meanwhile we are showing `how to do the change and what is the name of the variable` . Let me add a waring message in this section. Please feel free to change or suggest us. Thank you in advance.